sci-visualization/xgraph/xgraph-12.1-r4.ebuild

```diff
@@ -17,25 +17,24 @@ LICENSE="xgraph"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
 IUSE="examples"
-DEPEND="x11-libs/libSM
+RDEPEND="x11-libs/libSM
 	x11-libs/libX11"
-RDEPEND="${DEPEND}"
+DEPEND="$RDEPEND"
+
+PATCHES=( "${WORKDIR}"/debian/patches/debian-changes )
 
 src_prepare() {
-	epatch "${WORKDIR}"/debian/patches/debian-changes
-	rm configure.in Makefile.in
+	default
+	rm -f configure.in Makefile.in || die
```

Comment:
I'm not completely overseeing this, but do the debian patches create a new `configure.ac`? Otherwise you're killing the input file for autoconf